tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/Common/Response.cs

```diff
@@ -9,5 +9,7 @@ internal sealed class Response
         public HttpResponseMessage Result { get; set; }
 
         public async Task<JToken> ReadBodyAsJsonAsync() => JToken.Parse(await Result.Content.ReadAsStringAsync());
+
+        public async Task<string> ReadStringAsAsync() => await Result.Content.ReadAsStringAsync();
```

Comment:
Does it read better if applying a similar convention of naming as the method above e.g. ReadBodyAsStringAsync?

Reply:
👍